pom.xml

```diff
@@ -45,6 +45,32 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.deploy.skip>true</maven.deploy.skip>
   </properties>
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>xml-maven-plugin</artifactId>
+          <version>${xml-maven-plugin.version}</version>
+          <configuration>
+            <indentSize>2</indentSize>
+            <formatFileSets>
+              <formatFileSet>
+                <directory>${basedir}</directory>
+                <includes>
+                  <include>**/*.xml</include>
+                </includes>
+                <excludes>
+                  <exclude>**/.*</exclude>
+                </excludes>
+              </formatFileSet>
+            </formatFileSets>
+            <useDefaultFormatFileSet>false</useDefaultFormatFileSet>
```

Comment:
If `useDefaultFormatFileSet`  is true , xml-checkformat is executed twice(one use  DefaultFormatFileSet. two use Specified FormatFileSet).
In this case DefaultFormatFileSet is not necessary because Specified formatFileSet coverd undder `terasoluna-gfw` folder.